### DISCUSSION TITLE
docs/v1.13.x/advance/vpc-internal-lb/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#4809] [Announcements]
# docs/v1.13.x/advance/vpc-internal-lb/

None

https://kubeovn.github.io/docs/v1.13.x/advance/vpc-internal-lb/

**Comment by acechef** [comment]
hello，我现在已经通过共享底层 OVN方式将k8s和openstack进行打通，创建了nginx pod和vm间可以互相ping通。并创建了SwitchLBRule对nginx pod进行负载均衡，在相同的subnet下起了个busybox请求SwitchLBRule的vip可以成功访问nginx pod，但在vm处通过vip却访问不通，该怎么排查呢

**Reply by zbb88888** [reply]
提个 issue 吧，把虚拟机和 pod 的 lsp 信息 用 kubectl ko nbctl show 看下？ 社区有 ko trace 工具，可以跟踪下看看

**Reply by acechef** [reply]
好的
